### DISCUSSION TITLE
MVTLayer update url every n minutes

### DISCUSSION THREAD

**Discussion by am2222** [visgl/deck.gl#8992] [Q&A] [answered]
Hi,

So I have a MVTLayer added to the deck.gl map. This URL serves data from a URL that updates every 1 minute. So, technically, I need to update the layer's URL every minute to pass a timestamp to the URL parameters.

I was wondering if there are any best practices for such a thing? Is it better to hide older layers and add new layers or replace the existing layer?

Thank you!

**Comment by Pessimistress** [comment, marked as answer]
You can just change the `data` prop of your existing MVTLayer.

**Reply by am2222** [reply]
Thanks @Pessimistress
